test/jdk/java/foreign/TestLayouts.java

```diff
@@ -219,10 +220,10 @@ public void testSequenceOverflow() {
     @Test
     public void testSequenceLayoutWithZeroLength() {
         SequenceLayout layout = MemoryLayout.sequenceLayout(0, JAVA_INT);
-        assertEquals(layout.toString(), "[0:i4]");
+        assertEquals(layout.toString().toLowerCase(Locale.ROOT), "[0:i4]");
```

Comment:
Hello Per, I don't yet have experience with these APIs; do you think the implementation of `toString()` itself should perhaps be changed to return a consistent value that doesn't depend on the endianness? Or is it intentional that there is a dependency?

Reply:
I have seen this question several times in the past and it is indeed a valid one. It is like this by design and unfortunately, there will always be differences across platforms due to byte ordering, and mappings between C types and Java types.  As FFM acts like a glue between the native and Java worlds, there is always going to be a risk that one gets glue on one's fingers...